package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha --reporter spec"
```

Comment:
It is, I was testing with the --check-leaks option and by default I always add both. I removed check-leaks but just left in reporter. Makes it easy to change if needed. I can remove if you want.